src/test/java/mate/academy/dao/RoleDaoImplTest.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoImplTest extends AbstractTest {
+    private static final String USER_ROLE_NAME = "USER";
+    private static final String ADMIN_ROLE_NAME = "ADMIN";
+    private static final String WRONG_ROLE_NAME = "WRONG";
+    private static final Role USER_ROLE = new Role(Role.RoleName.USER);
+    private static final Role ADMIN_ROLE = new Role(Role.RoleName.ADMIN);
+    private RoleDao roleDao;
+
+    @BeforeEach
+    void setUp() {
+        roleDao = new RoleDaoImpl(getSessionFactory());
+        roleDao.save(USER_ROLE);
+        roleDao.save(ADMIN_ROLE);
+    }
+
+    @Test
+    void getRoleByName_User() {
```

Comment:
```suggestion
    void getRoleByName_ok() {
```
![image](https://user-images.githubusercontent.com/20326858/201911293-14c57543-4702-41d1-b9ae-144b02578f41.png)